examples/hacker_news/package.json

```diff
@@ -3,35 +3,30 @@
   "version": "1.0.0",
   "description": "Demonstrate a news articles with jotai, hit next to see more articles.",
   "keywords": [],
-  "main": "src/index.js",
+  "main": "src/index.tsx",
   "dependencies": {
-    "@react-spring/web": "9.0.0-rc.3",
-    "html-react-parser": "0.14.2",
-    "jotai": "^0.15.2",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
+    "@react-spring/web": "^9.2.3",
+    "html-react-parser": "^1.2.6",
+    "jotai": "^0.16.11",
```

Comment:
Thank you for your effort. Could you update this to jotai v1?

Reply:
And also other examples.

Reply:
I have done all.
@Aslemammad

Reply:
Thanks.